Automattic/jetpack
# VideoPress: show error notice when updating data video fails

Related to https://github.com/Automattic/jetpack/issues/27863

This PR shows a notice error when the user tries to update a video and something fails. It happens, for instance, when a video block is created sitting the video URL, and the user is not the video owner.

#### Changes proposed in this Pull Request:

* VideoPress: show error notice when updating data video fails

#### Other information:



#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the block editor
* Start to create a VideoPress video block
* Set the URL. Feel free to use `https://videopress.com/v/ezoR6kzb`
* Try to save the video title
* Confirm the app shows an error notice

<img width="998" alt="Screen Shot 2022-12-19 at 10 48 39" src="https://user-images.githubusercontent.com/77539/208409317-cf74d826-3ec9-4f62-9bd6-e0d4d343e53b.png">
